appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/PayaraMicro.java

```diff
@@ -1410,6 +1428,17 @@ private void scanArgs(String[] args) {
                     case "--disablePhoneHome":
                         disablePhoneHome = true;
                         break;
+                    case "--enableRequestTracing":
+                        enableRequestTracing = true;
+                        break;
+                    case "--requestTracingThresholdUnit":
```

Comment:
There should be some validation on the the units passed in on the command line to ensure they are valid units.